doc/design/s3-trailing-delete.md

```diff
@@ -0,0 +1,69 @@
+# **S3 Trailing Delete**
```

Comment:
I guess IDE did another misservice -- added not needed `**` around every section header. 
for consistency with other doc/design files and typical use of markdown - let's get rid of them.  I will push a commit